src/triage/cli.py

```diff
@@ -59,6 +60,81 @@ def stampdb(self, args):
               f"we think your results schema is version {revision} and are upgrading to it")
         stamp_db(revision, args.dbfile)
 
+
+@Triage.register
+class Experiment(Command):
+    """Validate and run experiments, manage experiment database"""
+
+    def __init__(self, parser):
+        parser.add_argument(
+            '-c', '--config',
+            type=argparse.FileType('r'),
+            help="config file for Experiment",
+            required=True
+        )
+        parser.add_argument(
+            '--n-db-processes',
+            type=int,
+            default=1,
+            help="number of concurrent database connections to use"
+        )
+        parser.add_argument(
+            '--n-processes',
+            type=int,
+            default=1,
+            help="number of cores to use"
+        )
+        parser.add_argument(
+            '--project-path',
+            type=str,
+            help="path to store matrices and trained models"
```

Comment:
does this have a default? (perhaps empty string works?)

anyway, `type=str` isn't necessary, since it's coming from the console.

Reply:
Maybe we pick something like the current working directory, or CWD+'experiments'.